schema/va-spec/profiles/diagnostic/diagnostic-source.yaml

```diff
@@ -0,0 +1,75 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/profiles/diagnostic/diagnostic-source.yaml"
+title: Diagnostic Definitions
+imports:
+  va.core: ../../core-im/core-im-source.yaml
+  gks.core: ../../../gks-common/core-source.yaml
+  gks.genes: ../../../gks-common/genes-source.yaml
+  gks.conditions: ../../../gks-common/conditions-source.yaml
+  gks.therapeutics: ../../../gks-common/therapeutics-source.yaml
+  catvrs: ../../../catvrs/catvrs-source.yaml
+  vrs: ../../../vrs/vrs-source.yaml
+
+namespaces:
+  va.core: ../../core-im/core-im.json#/$defs/
+  gks.core: ../../../gks-common/core.json#/$defs/
+  gks.genes: ../../../gks-common/genes.json#/$defs/
+  gks.conditions: ../../../gks-common/conditions.json#/$defs/
+  gks.therapeutics: ../../../gks-common/conditions.json#/$defs/
+  catvrs: ../../../catvrs.json#/$defs/
+  vrs: ../../../vrs/vrs.json#/$defs/
+
+$defs:
+  VariantDiagnosticStudy:
+    type: object
+    maturity: draft
+    inherits: va.core:VariantStudySummary
+    description: >-
+      A study summarization describing whether a variant is associated with a disease
+      (positive) or lack of a disease (negative).
+    properties:
+      type:
+        type: string
+        const: "VariantDiagnosticStudy"
+        default: "VariantDiagnosticStudy"
+        description: >-
+          MUST be "VariantDiagnosticStudy".
+      predicate:
+        extends: predicate
+        enum:
+          - positive
+          - negative
+      disease:
+        extends: object
+        oneOf:
+          - $refCurie: gks.conditions:Condition
+          - $refCurie: gks.core:IRI
+        description: >-
+          The disease that is evaluated for presence
```

Comment:
@ahwagner I wasn't quite sure what you meant by

> - Drop the `disease` property
> - Add a new `disease` property
>   - extends `object` (the VA notion of `object`)

I just assumed this meant adding `extends: object`. Can you clarify?

Reply:
You have it right! There was confusion about this on another thread (since a JSON {} struct is also called an object in JSON Schema).